cluster/gce/trusty/configure.sh

```diff
@@ -144,8 +144,17 @@ install_additional_packages() {
   fi
 }
 
-# Downloads kubernetes binaries and salt tarball, unpacks them, and places them
-# to suitable directories.
+# Retry a download until we get it.
+#
+# $1 is the file to creat
+# $2 is the URL to download
+download_or_bust() {
```

Comment:
I just remember there is a timeout for nodes to finish registering with master. It is not permanently trying, but trying with a minutes level timeout. In this manner, I think it makes sense. Will restore it to the original logic